packages/lesswrong/server/testingSqlClient.ts

```diff
@@ -150,59 +137,27 @@ export const killAllConnections = async (id?: string) => {
   }
 }
 
-const seedData = async <T extends {}>(collection: CollectionBase<any>, data: T[]) => {
-  // eslint-disable-next-line no-console
-  console.log(`Importing Cypress seed data for ${collection.options.collectionName}`);
-  await collection.rawInsert(data);
-}
-
 type DropAndCreatePgArgs = {
-  seed?: boolean,
   templateId?: string,
   dropExisting?: boolean,
 }
 
-export const dropAndCreatePg = async ({seed, templateId, dropExisting}: DropAndCreatePgArgs) => {
+export const dropAndCreatePg = async ({templateId, dropExisting}: DropAndCreatePgArgs) => {
```

Comment:
This file can probably be cleaned up further now it's only used by integration tests. We should probably also migrate the integration tests to use the new dockerized postgres DB which should speed them up considerably, but that can wait for a different PR.